iphone/Classes/TiUIiOSProxy.m

```diff
@@ -258,6 +261,20 @@ - (void)didReceiveMemoryWarning:(NSNotification *)notification
   [super didReceiveMemoryWarning:notification];
 }
 
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
```

Comment:
You wrap the method inside this `#if` guard, but register for the notification on all SDK versions. Could this lead to a crash when building with SDK < 13.0?

Reply:
That code snippet is also inside #if guard . See complete code-
`#if IS_SDK_IOS_13
    UIWindow *keyWindow = UIApplication.sharedApplication.keyWindow;
    CGRect frame = keyWindow.windowScene.statusBarManager.statusBarFrame;
    UIView *view = [keyWindow viewWithTag:TI_STATUSBAR_TAG];
    if (!view) {
      view = [[UIView alloc] initWithFrame:frame];
      view.tag = TI_STATUSBAR_TAG;
      [keyWindow addSubview:view];
     }
     view.frame = frame;
     view.backgroundColor = [[TiUtils colorValue:value] _color];
     [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(windowDidBecomeKey:) name:UIWindowDidBecomeKeyNotification object:nil];
 #endif`

Reply:
Duh, i totally missed that since it wasn't showing in the diff. Thanks for updating the constant though, consistency is always good!